amp/WEB-INF/src/org/digijava/module/aim/form/TeamAuditForm.java

```diff
@@ -21,7 +23,25 @@
     private int offset;
     private String useraction;
     private String frecuency;
+    private Collection<User> userList;
```

Comment:
we have the same list in AuditLoggerManagerForm, why might put it in the parent classs to reuse

Reply:
same all other methods